packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -70,6 +70,10 @@ const Widget _startSelectionOverlay = CupertinoPickerDefaultSelectionOverlay(cap
 const Widget _centerSelectionOverlay = CupertinoPickerDefaultSelectionOverlay(capStartEdge: false, capEndEdge: false);
 const Widget _endSelectionOverlay = CupertinoPickerDefaultSelectionOverlay(capStartEdge: false);
 
+/// Defines a function signature for creating a widget that serves as a selection overlay,
+/// given the current context, the selected item's index, and the total number of columns.
+typedef SelectionOverlayBuilder = Widget Function(BuildContext context, int index, int columnCount);
```

Comment:
Should we allow for this builder to return null? That way if somebody wanted to disable the overlay, like they can in CupertinoPicker, they can just have a function that returns null.

Reply:
Could they pass null to selectionOverlayBuilder instead of returning null, or is there a use case for both?